west.yml

```diff
@@ -222,7 +222,7 @@ manifest:
       groups:
         - hal
     - name: hal_silabs
-      revision: d191d981c4eb20c0c7445a4061fcdbcfa686113a
+      revision: 98b458d99b74083fce4c715fedcee45d388eb8b8
```

Comment:
You should point to the PR head directly, i.e. `revision: pull/x/head` where `x` is your PR number in the HAL.

Reply:
good idea, thx